Public/Src/Cache/ContentStore/BuildXL.Cache.ContentStore.dsc

```diff
@@ -22,16 +22,7 @@ namespace Default {
             {
                 subfolder: r`App`,
                 contents: [
-                    App.exe,
-                    // Back-Compat naming
-                    {
-                        file: App.exe.runtime.binary,
-                        targetFileName: "Microsoft.ContentStoreApp.exe",
-                    },
-                    {
-                        file: App.exe.runtime.pdb,
-                        targetFileName: "Microsoft.ContentStoreApp.pdb",
-                    }

```

Comment:
I'm not aware of anyone consuming these directly, though that doesn't mean it to be true. It would probably be good to double check with Office 4 Windows. Assuming they don't need any time to migrate anything it should be fine. Can you make sure it gets into the next release's release notes as breaking changes?

Reply:
Contacted OE, they have removed their final reference to `Microsoft.ContentStoreApp.exe` and believe that I'm clear to push this change.